modules/aws/vault/versions.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
```

Comment:
small thing: below we pin providers, here no. is this wanted? should we pin providers everywhere?